.prow/e2e-features.yaml

```diff
@@ -26,7 +26,7 @@ presubmits:
       preset-hetzner: "true"
       preset-openstack: "true"
       preset-vsphere: "true"
-      preset-kubevirt: "true"
+      preset-kubevirt-dc: "true"
```

Comment:
I would prefer to keep the previous `kubevirt-preset`  . Let's just update the already existing one with new credentials. Thanks.

Reply:
Should not we make sure that we have no issue on all repos (kkp/dashboard) before updating existing credentials ?
I would propose to do in sequence:
1) use `preset-kubevirt-dc` in all repos
2) when all are ok, change the `preset-kubevirt` with the new credentials...
Let me know what you think